buildscripts/buildvars-setup.cmd

```diff
@@ -41,9 +43,10 @@ if /i "%1" == "clean"   (set __CleanBuild=1&shift&goto Arg_Loop)
 
 if /i "%1" == "skiptests" (set __SkipTests=1&shift&goto Arg_Loop)
 if /i "%1" == "skipvsdev" (set __SkipVsDev=1&shift&goto Arg_Loop)
+if /i "%1" == "objwriter" (set __ObjWriterBuild=1&set "__ExtraMsBuildParams=%__ExtraMsBuildParams% /p:ObjWriterBuild=true"&shift&goto Arg_Loop)
 if /i "%1" == "/dotnetclipath" (set __DotNetCliPath=%2&shift&shift&goto Arg_Loop)
 
-if /i "%1" == "/officialbuildid" (set "__ExtraMsBuildParams=/p:OfficialBuildId=%2"&shift&shift&goto Arg_Loop)
+if /i "%1" == "/officialbuildid" (set "__ExtraMsBuildParams=%__ExtraMsBuildParams% /p:OfficialBuildId=%2"&shift&shift&goto Arg_Loop)
```

Comment:
Having extra build args is a good idea anyway, but are we building ObjWriter in official builds?

Reply:
I am not planning to enable building of ObjWriter in official builds as part of the set of changes that I am working on. I just want to be able to re-build it locally.

Eventually, we will want to enable it. I think we would also want to have our fork of LLVM so that official builds do not depend on LLVM fork that we do not manage.